Make reports error when No ELF files found

**Describe the bug**
Make reports an error when some configurations do not contain any ELF files.  It may be because they report "No ELF files found" rather than "RESULT: All xxx tests passed."  The Make error provides no more information and is misleading because the behavior is correct.

**To Reproduce**
Steps to reproduce the behavior:

In Makefile, set EXTENSIONS ?= ExceptionsSm
make spike

**Text output/Error Messages**
```
...
══════ spike-rv32-max ══════
  Running ELFs from: /home/harris/repos/riscv-arch-test/work/spike-rv32-max/elfs
  Using command: spike --isa=rv32imafdcbv_zicbom_zicboz_zicbop_zicfilp_zicond_zicsr_zicntr_zicclsm_zifencei_zihintntl_zihintpause_zihpm_zimop_zabha_zacas_zawrs_zfa_zfbfmin_zfh_zcb_zcmop_zbc_zkn_zks_zkr_zvfbfmin_zvfbfwma_zvfh_zvbb_zvbc_zvkg_zvkned_zvknha_zvknhb_zvksed_zvksh_zvkt_sscofpmf_smcntrpmf_sstc_svinval <elf_path>
  Summary available at: /home/harris/repos/riscv-arch-test/work/spike-rv32-max/summary.log

  RESULT: All 1 tests passed.

══════ spike-RVI20U32 ══════
  Running ELFs from: /home/harris/repos/riscv-arch-test/work/spike-RVI20U32/elfs
  Using command: spike --isa=rv32imafdc_zicclsm_zicsr_zifencei_zicntr_zca_zcf_zcd_zaamo_zalrsc <elf_path>
  Summary available at: /home/harris/repos/riscv-arch-test/work/spike-RVI20U32/summary.log
  No ELF files found

══════ spike-RVI20U64 ══════
  Running ELFs from: /home/harris/repos/riscv-arch-test/work/spike-RVI20U64/elfs
  Using command: spike --isa=rv64imafdc_zicclsm_zicsr_zifencei_zicntr_zca_zcd_zaamo_zalrsc <elf_path>
  Summary available at: /home/harris/repos/riscv-arch-test/work/spike-RVI20U64/summary.log
  No ELF files found
make: *** [Makefile:227: spike] Error 1
```

--- a/run_tests.py
+++ b/run_tests.py
@@ -198,7 +198,7 @@ def main() -> int:
     elf_files = sorted(elf_dir.rglob("*.elf"))
     if not elf_files:
         print(yellow("  No ELF files found"))
-        sys.exit(1)
+        sys.exit(0)
 
     partial_run_test = partial(run_test, command, log_dir, elf_dir, verbose=args.verbose)
 

